sdk/eventhub/event-hubs/src/eventDataBatch.ts

```diff
@@ -65,8 +70,16 @@ export class EventDataBatch {
    * @property Size of a batch of events.
    * @readonly
    */
-  get size(): number {
-    return this._size;
+  get sizeInBytes(): number {
+    return this._sizeInBytes;
+  }
+
+  /**
+   * @property Number of events in the batch.
+   * @readonly
+   */
+  get count(): number {
+    return this._count;
```

Comment:
Since the class shape changed, I believe the .md file for api would also have been updated. We should include that change